tests/e2e/e2e_bypassminfee_test.go

```diff
@@ -19,4 +29,16 @@ func (s *IntegrationTestSuite) testByPassMinFeeWithdrawReward() {
 	// fail
 	s.T().Logf("bypass-msg with non-zero coin not in the denom of global fee, fail")
 	s.execWithdrawAllRewards(s.chainA, 0, payee.String(), paidFeeAmt+photonDenom, true)
+
+	proposalCounter++
+	// change MaxTotalBypassMinFeeMsgGasUsage through governance proposal from 1_000_0000 to 1
+	s.govProposeNewMaxTotalBypassMinFeeMsgGasUsage(1, proposalCounter, submitter)
+
+	// fail
+	s.T().Logf("bypass-msg has zero coin and maxTotalBypassMinFeeMsgGasUsage set to 1, fail")
+	s.execWithdrawAllRewards(s.chainA, 0, payee.String(), "0"+uatomDenom, true)
+
+	// pass
+	s.T().Logf("bypass-msg has non zero coin and maxTotalBypassMinFeeMsgGasUsage set to 1, pass")
+	s.execWithdrawAllRewards(s.chainA, 0, payee.String(), paidFeeAmt+uatomDenom, false)
```

Comment:
maybe also do this withdraw also before proposing change maxTotalBypassMinFeeMsgGasUsage to 1, to make sure this pass also with maxTotalBypassMinFeeMsgGasUsage=1000000